### PR TITLE
add delete endpoint and test for 204. Further tests to be added when …

### PR DESCRIPTION
**Pull request by Roothebear**
…/api/comments endpoint created

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -368,6 +368,21 @@ describe("/api/articles", () => {
...
 
+// COMMENTS
+
+describe("/api/comments", () => {
+  // DELETE testing
+  describe("DELETE /api/comments/:comment_id", () => {
+    test("status: 204 comment is deleted", () => {
+      return request(app)
+        .delete("/api/comments/1")
+        .expect(204)
```
Comment (popatre) [inline __tests__/app.test.js:379]: Optionally, you could also check the comments array afterwards to see if the length is -1, what it was before.

File: controllers/controllers.js [hunk 2]
```diff
@@ -88,3 +89,13 @@ exports.patchArticleById = (req, res, next) => {
 };
 
 // DELETE CONTROLLERS
+
+exports.removeCommentById = (req, res, next) => {
+  deleteCommentById(req)
+    .then(() => {
+      res.status(204).end();
```
Comment (popatre) [inline controllers/controllers.js:96]: Can refactor this to res.sendStatus(204).  -will end it for you.

File: models/models.js [hunk 3]
```diff
@@ -165,3 +163,15 @@ exports.updateArticleById = (req) => {
...
 // DELETE MODELS
+
+exports.deleteCommentById = (req) => {
+  const { comment_id } = req.params;
+  return db
+    .query(`DELETE FROM comments WHERE comment_id = $1;`, [
+      comment_id,
+    ])
+    .then((result) => {
+      console.log(result)
```
Comment (popatre) [inline models/models.js:174]: Remove the console log.

File: controllers/controllers.js (hunk 2 above)
Comment (Roothebear) [reply controllers/controllers.js:96]: Ah, nice, thank you!

**Comment by Roothebear** [comment]
Thank you for the feedback

**Pull request merged**